src/app/complex/complex-results/complex-paginator/complex-paginator.component.html

```diff
@@ -1,12 +1,15 @@
 <div>
-  <h5 class="text-center">{{currentPageIndex}} of {{lastPageIndex}}</h5>
+  <h5 class="text-center">{{ currentPageIndex }} of {{ lastPageIndex }}</h5>
   <ul class="pagination text-center" role="navigation" aria-label="Pagination">
+    <li class="arrow unavailable" *ngIf="currentPageIndex != 1" aria-disabled="true"><a
+      (click)="getFirstPage()">&laquo; First</a></li>
```

Comment:
Currently we are using `&laquo;` for first and previous page and `&raquo` for next and last page, meaning we are using the same icon for first and previous, and the same icon form next and last.

We can change it to:
- First: `&Lang;`
- Previous: `&lang;`
- Next: `&rang;`
- Last: `&Rang;`

That way we have different icons for each button.